cryptography/src/jsMain/kotlin/com/wire/kalium/cryptography/ProteusClient.kt

```diff
@@ -0,0 +1,115 @@
+package com.wire.kalium.cryptography
+
+import com.wire.kalium.cryptography.exceptions.ProteusException
+import com.wire.kalium.cryptography.externals.Cryptobox
+import com.wire.kalium.cryptography.externals.IdentityKey
+import com.wire.kalium.cryptography.externals.MemoryEngine
+import com.wire.kalium.cryptography.externals.PreKeyBundle
+import io.ktor.util.InternalAPI
+import io.ktor.util.decodeBase64Bytes
```

Comment:
So this was an internal API,  is anyone aware of another Base64 implementation which works in common?

Reply:
`ktor-utils` is available as a standalone library too. I'm not sure if maybe we could just use it on `commonMain` and use it for all platforms. Maybe creating a `Base64Encoder` for all platforms to use? 

In any case, it works!